spark/src/test/scala/org/apache/comet/exec/CometJoinSuite.scala

```diff
@@ -251,4 +251,18 @@ class CometJoinSuite extends CometTestBase {
       }
     }
   }
+
+  // TODO: Add a test for CartesianProductExec with join filter after new DataFusion release
```

Comment:
Add a github issue for the followup and refer to it here?

Reply:
fine